libvast/vast/logger.hpp

```diff
@@ -42,6 +42,12 @@
 #include "vast/detail/logger.hpp"
 #include "vast/detail/logger_formatters.hpp"
 
+#if FMT_VERSION >= 80000 // {fmt} 8+
+#  define VAST_FMT_RUNTIME(...) ::fmt::runtime(__VA_ARGS__)
```

Comment:
Since it doesn't appear in the upstream fmt docs, let's add a comment here to explain that `fmt::runtime()` is a function that can be used to enforce runtime evaluation of the format string for the case where it is not known at compile time.

Reply:
Added a comment.